packages/os/os.spec

```diff
@@ -299,6 +304,11 @@ Summary: XFS progs cli
 %description -n %{_cross_os}xfscli
 %{summary}.
 
+%package -n %{_cross_os}licenses
```

Comment:
You would need to add a `Requires` on `%{_cross_os}licenses` to have this package actually be installed (in the same way that we require e.g. `%{_cross_os xfscli}`)

I see now that we handle attributions in the top-level packaging below, I think it makes sense to add our licenses to there rather than create a subpackage.